specs/eip4844/polynomial-commitments.md

```diff
@@ -336,35 +379,38 @@ def blob_to_kzg_commitment(blob: Blob) -> KZGCommitment:
 #### `verify_kzg_proof`
 
 ```python
-def verify_kzg_proof(polynomial_kzg: KZGCommitment,
+def verify_kzg_proof(commitment_bytes: Bytes48,
                      z: Bytes32,
                      y: Bytes32,
```

Comment:
Shouldn't `z` and `y` be `z_bytes` and `y_bytes`?

Reply:
We decided to do this rename in the future.